scripts/migrate-black.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+# check out every commit added by the current branch, blackify them,
+# and generate diffs to reconstruct the original commits, but then
+# blackified
+import os
+import sys
+from subprocess import check_output, run, Popen, PIPE
+
+
+def git(*args):
+    return check_output(["git"] + list(args)).decode("utf8").strip()
+
+
+def blackify(base_branch, black_command):
+    current_branch = git("branch", "--show-current")
+
+    if not current_branch or base_branch == current_branch:
+        print("You need to check out a feature brach to work on")
+        return 1
+
+    if not os.path.exists(".git"):
+        print("Run me in the root of your repo")
+        return 1
+
+    merge_base = git("merge-base", "HEAD", base_branch)
+    if not merge_base:
+        print("Could not find a common commit for current head and %s" % base_branch)
+        return 1
+
+    commits = git(
+        "log", "--reverse", "--pretty=format:%H", "%s~1..HEAD" % merge_base
+    ).split()
+    for commit in commits:
+        git("checkout", commit, "-b" "%s-black" % commit)
+        check_output(black_command, shell=True)
```

Comment:
```suggestion
        import shlex
        check_output(shlex.split(black_command))
```
Shouldn't need shell=True here ...

Reply:
the idea was to allow things like `black .; isort .` here